wpilibj/src/athena/java/edu/wpi/first/wpilibj/ADXL345_SPI.java

```diff
@@ -53,7 +53,7 @@
     @SuppressWarnings("MemberName")
     public final byte value;
 
-    private Axes(byte value) {
+    Axes(byte value) {
```

Comment:
This is actually a change of accessibility. I don't think this is what we want here.